dash/dash-renderer/src/actions/callbacks.ts

```diff
@@ -274,8 +272,6 @@ function handleClientside(
             throw e;
         }
     } finally {
-        delete dc.callback_context;
```

Comment:
Oh actually, we might want to keep the delete here too - so that if there's an error in a sync callback we still delete the context.

Reply:
What would happen if we kept the deletion here and let it be stale within the async part of the callbacks? The guidelines is to not use it in the async portions of your callbacks anyway.

Reply:
My worry is that it would mostly work, so users would come to rely on it, but then there would be intermittent problems, for example if there's a callback linked to a `dcc.Interval` or if the user triggers two actions in quick succession, that would be very hard to debug. Better to make it never work than fail sporadically.

Reply:
Ah yes, that would definitely be the case. Let's have it both places for now and it can be removed in future if a way to have a stable context is found.